src/pages/similarity-indexes.html

```diff
@@ -96,92 +96,65 @@ <h3 class="mb-0"><span class="collapsible-heading" data-toggle="collapse" data-t
 		</div>
 	</div>
 
-	<div class="card mb-2 index-search-panel" ng-hide="selected === undefined || selected.type == 'textLiteral'">
+	<div class="card mb-2 index-search-panel" ng-hide="selected === undefined || similarityIndexType.TEXT_LITERAL === selected.type">
 		<div class="card-block">
 			<h3 class="selected-index">{{'search.in.index' | translate}} <strong>{{selected.name}}</strong></h3>
 
-			<div ng-show="selected.type == 'text'">
+			<div ng-show="similarityIndexType.TEXT === selected.type">
 				<div class="d-inline-block">
 					<strong>{{'similarity.search.type' | translate}} </strong>
 					<div class="btn-group">
 						<div class="dropdown">
 						  <button class="btn btn-link dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-						    <span ng-if="searchType == 'searchTerm'">{{'term' | translate}}</span>
-						    <span ng-if="searchType == 'searchDocumentID'">{{'document' | translate}}</span>
+						    <span ng-if="similaritySearchType.SEARCH_TERM === searchType">{{'term' | translate}}</span>
+						    <span ng-if="similaritySearchType.SEARCH_DOCUMENT === searchType">{{'document' | translate}}</span>
 						  </button>
 						  <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-						    <a class="dropdown-item" ng-click="searchType = 'searchTerm'">{{'term' | translate}}</a>
-						    <a class="dropdown-item" ng-click="searchType = 'searchDocumentID'">{{'document' | translate}}</a>
+						    <a class="dropdown-item" ng-click="searchType = similaritySearchType.SEARCH_TERM">{{'term' | translate}}</a>
```

Comment:
Personally I don't like having statements in the view. Can you extract this and use a function in the ctrl?

Reply:
Yes, I will do it in some next request. After discussion with @svilenvelikov we decided to make the changes on small pieces to be easy for review. I will make some other changes and then will extract them in the ctrl function.